locale/en/corteza-webapp-compose/notification.yaml

```diff
@@ -18,6 +18,9 @@ chart:
   saveFailed: Could not save this chart
   saved: Chart saved
   unsupportedRenderer: Unsupported renderer
+  liveChart:
```

Comment:
Hmm its a bit strange of a place to put this (notification). I think it belongs more in the blocks translations. 

I suggest you fix the namespace: 'notifications' to 'block' so its handled same as other blocks. And also make sure that older translations work!

So this key would be in block.chart.filter